src/backend/drm/atomic/surface.rs

```diff
@@ -1032,3 +1032,16 @@ impl<A: AsRawFd + 'static> RawSurface for AtomicDrmSurface<A> {
         RawSurface::page_flip(&*self.0, framebuffer)
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::AtomicDrmSurface;
+    use std::fs::File;
+
+    fn is_send<S: Send>() {}
+
+    #[test]
+    fn surface_is_send() {
+        is_send::<AtomicDrmSurface<File>>();
+    }
+}
```

Comment:
These tests make sure, that we do not violate our new `Send` feature unknowingly during any refactoring or feature development. It makes sure, that for a type we know is `Send` (e.g. File), the Surface is `Send` as well.